From 68bd98eb4144ce1f784e9c7aee4735a3f668453e Mon Sep 17 00:00:00 2001
From: Jim Ramsay <jim_ramsay@dell.com>
Date: Mon, 1 Nov 2010 10:11:30 -0400
Subject: [PATCH 4/4] Do not force -O2 in CFLAGS

Only uses -O2 if the user has not specified any CFLAGS in the
environment.
---
 Makefile |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 02ca376..c4ca35c 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,8 @@ GKRELLM_INCLUDE= -I/usr/local/include
 
 INSTALL=install -c -m 755
 
-CFLAGS += -O2 -Wall -fPIC $(GTK_INCLUDE) $(IMLIB_INCLUDE) $(GKRELLM_INCLUDE)
+CFLAGS ?= -O2
+CFLAGS += -Wall -fPIC $(GTK_INCLUDE) $(IMLIB_INCLUDE) $(GKRELLM_INCLUDE)
 LIBS = $(GTK_LIB) $(IMLIB_LIB)
 LDFLAGS += -shared
 
-- 
1.7.3.1

